Inventory/Items/AppleItem.gd

```diff
@@ -6,6 +6,9 @@ func _init():
 
 # apple is illegal only temporally as a test item
 func getVisibleName():
+	if(GM.pc != null):
+		if(GM.pc.hasPerk(Perk.FertilityProudMom) && getWearer() == GM.pc):
+			return "Tasty Apple"
```

Comment:
I was thinking about making a separate "legal" apples, but that made no sense to me because stolen or not apples look the same. But I can make Nurse give the player apple juice instead.

Reply:
if isNotStolen:
desc += "\nThis particular one has AlphaCorp's sticker of approval"

could be something like that.